src/RoadMarking/RoadMarking.tsx

```diff
@@ -0,0 +1,53 @@
+import { Line } from "react-konva";
+import React from "react";
+import { RoadMarkingProps } from "./RoadMarking.types";
+
+const RoadMarking = (
+  {
+    points,
+    dash = [],
+    width = 0.15,
+    type = "single",
+    color = "#ffffff"
+  }: RoadMarkingProps,
+  key: React.Key
+) => {
+  switch (type) {
+    case "double":
+      // plot symmetrical double lines
+      return (
+        <>
+          <Line
+            key={key}
+            strokeWidth={width * 2.5}
+            scaleY={-1}
+            dash={dash}
+            stroke={color}
+            points={points}
+          />
+          <Line
+            key={`${key}_cutout`}
+            strokeWidth={width * 0.5}
```

Comment:
The weird looking area between the lines is still there. I think it is because there is a transition from the end of the double lines to transparent, and if there is not enough space for the transition to happen, it gets blurry.

I would suggest to increase this value to 0.8 (barely visible) or 0.9, to avoid having this. I guess it depends whether that still looks acceptably close, I am just afraid of having an ugly blurred area between the lines.

Reply:
I don't think this is the solution as it would only solve the blurred lines at a specific combination of `width` and `scale`. Since the end goal is to allow the user to zoom in or zoom out, the blurry border would be notable at some zoom.

Furthermore, over a dark background like a road the blurred bit is barely visible. Additionally, I have tried changing the `globalCompositeOperation` to `xor` this results in the same behavior. I have also tried to make the 'middle' line a white line on top, same issue persists. I just don't think it is an issue since the user can zoom.

Reply:
My main concern is that once this gets to RESULT, the user will probably zoom onto the vehicle, to see events etc. Then this would be easily noticeable. I am not sure it is best to just leave it as is, but I guess it is a question of whether this is an important enough issue or not.

Since we seem to be on a deadlock on this one, it might be best to ask a third opinion on this one.

Reply:
The point I have been trying to make is specifically when you zoom this issue won't happen. You're using browser zoom, when the user zooms in the roadviewer we change the scale of the internal components of the canvas, thus this won't be visible.

Reply:
https://github.com/IPG-Automotive-UK/react-ui/assets/67595415/93400112-c2d0-4fe2-84a3-a523a683201f

Reply:
@jegasriskantha as DRI for scene could you advise please?

Reply:
The discussion ended up with it being OK.